changes/14887-update-cpe-nvd-api

```diff
@@ -0,0 +1,2 @@
+Updating CPE generator to use new NVD API.
```

Comment:
I believe we don't need a changes file (given that this change is not user facing).